### ISSUE TITLE
self - collision of cloth and response to avoid

### ISSUE THREAD

**Issue by jjqcat** [InteractiveComputerGraphics/PositionBasedDynamics#82]
I want to use it for clothing simulation, how to avoid self-collide?

**Comment by janbender** [comment]
Currently, there is no detection for self-collisions implemented. The only solution would be to implement an own collision detection or to integrate a existing one.

**Comment by jjqcat** [comment]
thanks for quick reply,are you planning to implement self-collision avoiding in the future version?

**Comment by janbender** [comment]
Yes, but probably in far future. Currently, we have not enough time for that.
